framework/auth/views.py

```diff
@@ -443,7 +455,7 @@ def unconfirmed_email_add(auth=None):
     }, 200
 
 
-def send_confirm_email(user, email):
+def send_confirm_email(user, email, renew=False):
```

Comment:
For CR: use `renew` to generate a new email verification token when user choose to resend confirmation email